rules/group.go

```diff
@@ -98,7 +98,9 @@ type GroupOptions struct {
 func NewGroup(o GroupOptions) *Group {
 	opts := o.Opts
 	if opts == nil {
-		opts = &ManagerOptions{}
+		opts = &ManagerOptions{
+			NameValidationScheme: model.UTF8Validation,
```

Comment:
When making changes to Prometheus, I ran into a lot of problems requiring non-empty struct initialization. There would be some test somewhere or some importing project that sets the struct via empty `{}` initializer, and they would be broken.  In a recent case, @krajorama spent a long time trying to track down a [breakage](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/40788) that turned out to be due to a default-constructed config struct in otel collector ([the fix](https://github.com/prometheus/prometheus/pull/16751)).    So I believe using New functions (preferred) or Validate functions should always be used for complex structs.